assets/wizards/setup/views/newsroom/style.scss

```diff
@@ -0,0 +1,13 @@
+.newspack-setup-wizard_newsroom-screen_plugin_group {
+	.muriel-checkbox {
+		margin-left: 2em;
+		margin-right: 2em;
+		float: left;
+    	min-width: calc( 50% - 4em );
```

Comment:
Mixed spaces and tabs on this line.

Reply:
https://github.com/Automattic/newspack-plugin/pull/114/commits/4bac47f93e8ebdce0927666070714808fd5fe9ce